projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Contacts.php

```diff
@@ -3251,7 +3251,7 @@ public function addUpdateContact($args=array()){
                             'customerExtraMeta'=>$confirmedExtraMeta #} This is the "extraMeta" passed (as saved)
                         ));
 
-						$dataArr['id'] = $newID; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect
+						$dataArr['ID'] = $newID; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect
```

Comment:
We have to test if this change could affect the normal operation of Jetpack CRM

Reply:
All is good here. It's only used for notifications and converted by `tidy_contact` call from `ID` to `id`